packages/web/src/components/FatalErrorBoundary.tsx

```diff
@@ -27,25 +28,25 @@ class InfallibleErrorBoundary extends React.Component<
 
 type PropsFatalErrorBoundary = {
   children?: React.ReactNode
-  page: React.ComponentType
+  page: React.ComponentType<{ error?: Error }>
 }
 
 class FatalErrorBoundary extends React.Component<
   PropsFatalErrorBoundary,
   State
 > {
-  state = { hasError: false }
+  state = { hasError: false, error: undefined }
 
-  static getDerivedStateFromError() {
-    return { hasError: true }
+  static getDerivedStateFromError(error: Error) {
+    return { hasError: true, error }
   }
 
   render() {
     const { page: Page } = this.props
     if (this.state.hasError) {
       return (
         <InfallibleErrorBoundary>
-          <Page />
+          <Page error={this.state.error} />
         </InfallibleErrorBoundary>
       )
     }
```

Comment:
Ensures that the highest level app error boundary is passed through too the users's error page.